tests/fields.py

```diff
@@ -367,7 +367,7 @@ def test_text_coercion(self):
             form.a(),
             '''<ul id="a">'''
             '''<li><input id="a-0" name="a" type="radio" value="True"> <label for="a-0">yes</label></li>'''
-            '''<li><input checked id="a-1" name="a" type="radio" value="False"> <label for="a-1">no</label></li></ul>'''
+            '''<li><input id="a-1" name="a" type="radio" value="False"> <label for="a-1">no</label></li></ul>'''
```

Comment:
Instead of changing the result, we could also pass `default=False` to the `RadioField` constructor.

Reply:
Why would it not be appropriate?

Reply:
There's no reason to select the second choice:

```
make_form(a=RadioField(choices=[(True, 'yes'), (False, 'no')], coerce=coerce_func))
```

Having none of the options pre-selected seems to be the better choice.